src/js/common/helpers/api.js

```diff
@@ -37,5 +37,6 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
 
       return data;
     })
-    .then(success, error);
+    .then(success)
+    .catch(error);
```

Comment:
Is there a reason for this change? It behaves the same I assume, but might just be more descriptive of what's actually happening?

Reply:
Yeah, this one was tricky. Basically, if the `success` function threw an error for whatever reason, it wouldn't be caught by `error`. I was running into that while trying to mock out the `fetch`.

Reply:
Ah okay, good catch!